packages/wallet/src/domain/commitments/__tests__/index.ts

```diff
@@ -116,6 +131,51 @@ export const ledgerChannel = {
 };
 export const ledgerId = channelID(ledgerChannel);
 
+export const threeWayLedgerChannel = {
```

Comment:
I'm following the newer pattern of using the test helper functions that construct `channel`s for you as opposed to using static test data.

I think the big benefit is that a test can create the type of `channel` it needs as opposed to relying on existing test data working for its use case. This is a less of an issue with joint channels since they're used on in a couple of places but I still think it makes sense to follow the same pattern.

Reply:
I agree. I wonder if we should
A. Migrate towards that pattern
B. Consolidate the test data into one place

I would have used this approach, except I found the static test data in another file.